### ISSUE TITLE
Struggeling to sign app with Frameworks

### ISSUE THREAD

**Issue by aluedeke** [indygreg/apple-platform-rs#163]
i am getting the following error when resigning an app bundle which contains Frameworks:

```
signing bundle at agent/agent-app.app/Frameworks/FlyingSocks.framework into agent/agent-app.app/Frameworks/FlyingSocks.framework
found an unversioned framework; signing like normal
collecting code resources files
Error: I/O error: No such file or directory (os error 2)
```
The error happens [here](https://github.com/indygreg/apple-platform-rs/blob/b59c6bda65df29bca5bda7bd80b7569637fd813c/apple-codesign/src/bundle_signing.rs#L623).

```
    /// Obtain the path to the `Info.plist` file.
    pub fn info_plist_path(&self) -> PathBuf {
        match self.package_type {
            BundlePackageType::App | BundlePackageType::Bundle => self.resolve_path("Info.plist"),
            BundlePackageType::Framework => self.root.join("Resources").join("Info.plist"),
        }
    }
```
The returned path is always `Resources\Info.plist`. But my frameworks always have their Info.plist in the root.

![Screenshot 2024-10-22 at 14 11 34](https://github.com/user-attachments/assets/632cdeed-c4cb-4120-b3d5-774e5d3dacdf)

Why would you search in Resources instead of the root folder?

**Comment by aluedeke** [comment]
@indygreg let me know if you need more to repro the issue

**Comment by indygreg** [comment]
I believe this is fixed by #157. Please reopen if you can reproduce on `main`. I'll likely publish a new release in the next few hours with this fix.
